package.json

```diff
@@ -67,6 +67,7 @@
 		]
 	},
 	"prettier": {
+		"endOfLine": "auto",
```

Comment:
so it passes on windows. By default prettier reads from `.editorconfig`. We could add a `.gitattributes` file as well

Reply:
alternatively, we could have separate jobs for tests (all OSes and node versions) and lint/typescript (LTS on ubuntu)

Reply:
I'm a bit confused. Is prettier using CRLF on Windows, despite `.editorconfig` currently containing `end_of_line = lf`? Or where are the CRLFs coming from? (My pref is to use LF on all platforms.)

Reply:
GitHub by default checks out with crlf. We can add a gitattributes file as well

Reply:
See https://github.community/t5/GitHub-Actions/git-config-core-autocrlf-should-default-to-false/td-p/30445

Reply:
I'm +1 on using `.gitattributes`. autocrlf is evil :)